ci-operator/config/openshift/kubernetes/openshift-kubernetes-master.yaml

```diff
@@ -197,10 +197,14 @@ tests:
   steps:
     cluster_profile: gcp
     workflow: openshift-kubernetes-e2e-gcp
+- as: e2e-aws
+  steps:
+    cluster_profile: aws
+    workflow: openshift-e2e-aws-loki
 - as: e2e-gcp
   steps:
     cluster_profile: gcp
-    workflow: openshift-e2e-gcp
+    workflow: openshift-e2e-gcp-loki
```

Comment:
why is loki removed? @ingvagabund

Reply:
It has never been in openshift/kubernetes repo yet